app/src/main/java/com/activities/AddFaceActivity.java

```diff
@@ -67,7 +68,7 @@
             result -> {
                 // Process picked image
                 if (result.getResultCode() == Activity.RESULT_OK && result.getData() != null) {
-                    processPhoto(resolveContentToBitmap(result.getData().getData(), this));
+                    processPhoto(loadBitmapFromUri(result.getData().getData(), this));
```

Comment:
Shouldn't we check null return value here?

Reply:
+1